lib/index.js

```diff
@@ -389,7 +390,15 @@ function handlePropertyValue(schema, value, prop, definition) {
       typeof value === 'number' ||
       typeof value === 'string') &&
     safeProtocol(schema, value, prop) &&
-    (definition.length === 0 || definition.includes(value))
+    (definition.length === 0 ||
+      definition.some((allowed) =>
+        typeof value === 'string' &&
+        allowed &&
+        typeof allowed === 'object' &&
+        'flags' in allowed
+          ? allowed.test(value)
+          : allowed === value
```

Comment:
```suggestion
      definition.some((allowed) =>
        typeof allowed === 'string'
          ? allowed === value
          : allowed.test(String(value))
```

How about this? Should do the same, and add “some” support for numbers, booleans.

Reply:
Need to check if `allowed` is regex before calling `test`.

How about that?

```js
definition.some((allowed) =>
  allowed && typeof allowed === 'object' && 'flags' in allowed
    ? allowed.test(String(value))
    : allowed === value

Reply:
Why? This checks the inverse: that something is a string. It being a regex is implied otherwise?

Reply:
This is also fine to me tho

Reply:
> Why? This checks the inverse: that something is a string. It being a regex is implied otherwise?

For TS, `allowed` can also be `boolean` and `number`

Reply:
ah, ok